style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 20px;
+  background-color: #f5f5f5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+h1 {
+  margin-top: 0;
+  color: #333;
+}
+
+form {
+  margin-bottom: 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+h2 {
+  margin-bottom: 10px;
+  color: #333;
+}
+
+ul {
+  list-style-type: none;
+  padding: 10px;
+}
+
+li {
+  border-bottom: 1px solid #ccc;
+  padding: 10px;
+  display: flex;
+  gap: 20px;
+}
+
+nav {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  border: 1px solid #1c1919;
+  padding: 8px;
+  box-shadow: 0 -2px 8px rgba(0, 0, 0, 0.2);
+}
+
+nav ul {
+  width: inherit;
+  list-style: none;
+  display: flex;
+  justify-content: flex-end;
+  padding: 0;
+}
+
+nav li {
+  margin: 0 10px;
+}
+
+.content-section {
+  display: none;
+}
+
+.content-section.show {
+  display: block;
+  margin-top: 10vh;
+  width: 88.5vw;
+}
+
+.content-section > h2 {
+  margin-left: 35vw;
+  margin-bottom: 3vh;
+}
+
+.content-section > p {
+  margin-left: 35vw;
+}
+
+.content-section > h1 {
+  margin-left: 36vw;
+}
+
+main {
+  margin-top: 2rem;
+}
+
+footer {
+  background-color: #f4f1f1;
+  padding: 8px;
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+  width: 90%;
+  border: 1px solid #1c1919;
+  box-shadow: 0 -2px 8px rgba(0, 0, 0, 0.2);
+}
+
+#bookListItems {
+  border: 2px solid black;
+  width: 86.8vw;
+  margin: auto;
+}
+
+.single-book {
+  /* border: 1px solid black; */
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: